### DISCUSSION TITLE
Pages not getting deployed

### DISCUSSION THREAD

**Discussion by rajtilakjee** [community/community#57459] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello everyone,

My GitHub pages were being deployed fine till last night. However, after I added some documentation, it stopped getting deployed. I am using Material for Mkdocs on a Windows 11 system. Here's the link to my repo: https://github.com/rajtilakjee/SignLink

It seems the gh-pages branch is not updating. Not sure if this helps but if I go to my homepage (https://rajtilakjee.github.io/SignLink/) and click on the pencil shaped edit button, it tries to open https://github.com/rajtilakjee/SignLink/edit/master/docs/index.md which doesn't exist. However, if I change master to main, I can see the intended page.

I am at a loss and not sure what to do. Please help!

**Comment by rajtilakjee** [comment]
Update: Just now a deployment was successful. However, even though the home page looks updated, the License page shows 404 error. I am really not sure what is causing this issue.

**Comment by airtower-luna** [comment, marked as answer]
The links for your license pages seem wrong. The "SignLink" one points at `https://rajtilakjee.github.io/SignLink/license/signlink-license.md`. But there's no such file in your repository (on the `gh-pages` branch). What exists is a `license/signlink-license/` *directory* holding an `index.html` file. You'll need to either point the link at the directory, or place the file where the link points to.

**Reply by rajtilakjee** [reply]
It was because of the outage yesterday. It's working fine right now.
